apps/studio/src/server/modules/resource/resource.router.ts

```diff
@@ -33,11 +33,12 @@ export const resourceRouter = router({
     }),
   getFolderChildrenOf: protectedProcedure
     .input(getChildrenSchema)
-    .query(async ({ input: { resourceId, cursor: offset, limit } }) => {
+    .query(async ({ input: { siteId, resourceId, cursor: offset, limit } }) => {
```

Comment:
do we need to update the callsites? or we have always been passing it in but not using it lol!

Reply:
passing in but ont used HAHAHAH

Reply:
<img width="718" alt="image" src="https://github.com/user-attachments/assets/1e820648-4b80-404a-9bed-67cf7b593fd2">

in `MoveResourceModal`, only place where `getFolderChildrenOf` is used